### ISSUE TITLE
Help  or Feature Request -- Overlay External Information on Web Page

### ISSUE THREAD

**Issue by beckrd** [aaronwmorris/indi-allsky#214]
I would like to overlay information from a python script I have running on the pi.  The script monitors the CPU and enclosure temperature (using a DHT22) and runs a fan if temperature thresholds are met.  With Thomas Alquin's Allsky, I put the information in an html file and used a php include to add this information to his "public.php" page.  

I could also envisage grabbing ambient conditions from my Davis weather gauge (already captured for INDI use).  Any recommendations or feature enhancement would be appreciated.

**Comment by aaronwmorris** [comment]
I will take a look to see how they do that and see if I can do something comparable.

**Comment by aaronwmorris** [comment]
Can you post the example code or a screen shot to see how it is presented?

**Comment by beckrd** [comment]
Here is my added line to public.php:

` <?php include("/home/pi/allsky/temp+fan.html"); ?>`

It's placed between the </script> and the </body> lines near the end of the file.

With this in place, the public.php displays the image with the text of the file displayed as follows

Date
CPU Temperature = xx.x C
Case Temperature = xx.x C
Fan is (on or off, depending on the temperatures)

**Comment by aaronwmorris** [comment]
What do you think about having the data in a text file and including each line in the status under the info at the bottom right?
![Screen Shot 2022-06-03 at 11 09 12 PM](https://user-images.githubusercontent.com/17464290/171977751-d6cd3949-e0c2-4e33-91e1-7d5695a009e9.png)

**Comment by beckrd** [comment]
[AllSky_Public_Page.pdf](https://github.com/aaronwmorris/indi-allsky/files/8837569/AllSky_Public_Page.pdf)
I restarted TA's Allsky and printed out the page displayed  by my custom public.php.

Your suggestion of adding user content to the info at the bottom would likely be a good idea for other users as well.

**Comment by aaronwmorris** [comment]
Merged #216 to include data in the web view.  The config option is the config form.

The text file can contain HTML tags if you desire.

```
CPU Temp: 20.1C
Case Temp: <span style="color:#ff0000;">800C</span>
Fan: Off
```

![Screen Shot 2022-06-04 at 11 57 48 AM](https://user-images.githubusercontent.com/17464290/172013852-71d1af37-818d-4cea-a4ba-183bee24a87b.png)
